docs/mdsource/parameterised.source.md

```diff
@@ -46,11 +46,11 @@ snippet: xunitMemberData
 
 ### Complex MemberData
 
-xUnit only exposes parameter information when the types certain types. For unknown types the parameter information cannot be retrieved from the xUnit context, and instead the parameters need to be explicitly passed in. This is done by calling `UseParameters()`.
```

Comment:
This is the part that made me think that xUnit handled parameter names for some types but needed `UseParameters()` for complex types.